bedrock/settings/base.py

```diff
@@ -260,13 +262,16 @@ def JINJA_CONFIG():
 PIPELINE_COMPILERS = (
     'pipeline.compilers.less.LessCompiler',
 )
-PIPELINE_LESS_BINARY = path('node_modules', 'less', 'bin', 'lessc')
```

Comment:
these bin paths are unlikely to change as well right? assuming we stick with installing via NPM.

Note: as soon as we're in a better way with deployment I want to use npm directly and delete the `node_modules` dir from the repo. We'd use [npm-lockdown](https://github.com/mozilla/npm-lockdown) of course.